date_range/README.rst

```diff
@@ -91,6 +91,8 @@ Contributors
 ------------
 
 * Laurent Mignon <laurent.mignon@acsone.eu>
+* Andrea Stirpe <a.stirpe@onestein.nl>
```

Comment:
IMO this kind of changes doesn't justify 2 new contributors. But thank you for the migration.